### PR TITLE
DEC-635 Migrate remaining exhibit methods to collection model

### PR DESCRIPTION
**Pull request by rfox2**
What: Moved remaining exhibit methods to the collection model.
How: Methods were moved and unit tests were updated. Also decided to comment out the exhibit model methods. Other unit tests had to be commented out because they referenced the old methods that are no longer instantiated. Those unit tests will be deleted once we remove the exhibit model from the code.

### CODE REVIEW

File: app/models/exhibit.rb [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+  #
+  # has_paper_trail
+  #
+  # def items_json_url
+  #   "/api/collections/#{collection_id}/items.json?include=image"
+  # end
+  #
+  # def item_json_url(item_id)
+  #   "/api/collections/#{collection_id}/items/#{item_id}.json?include=image"
+  # end
```
Comment (jonhartzler) [inline app/models/exhibit.rb]: I would say just delete this we have source control.

File: app/models/exhibition.rb [hunk 2]
```diff
@@ -66,17 +66,16 @@ def collection
     @collection ||= get_collection
   end
 
-  # rubocop:disable Metrics/AbcSize
   def save!
     ActiveRecord::Base.transaction do
-      exhibit.collection = collection
+      # exhibit.collection = collection
```
Comment (jonhartzler) [inline app/models/exhibition.rb]: These as well.

**Comment by danthewolfe** [comment]
:+1:

File: app/models/exhibit.rb (hunk 1 above)
Comment (rfox2) [inline app/models/exhibit.rb]: OK

File: app/models/exhibition.rb (hunk 2 above)
Comment (rfox2) [inline app/models/exhibition.rb]: Removed

**Pull request merged**